packages/node/src/env.test.ts

```diff
@@ -1,21 +1,70 @@
-import { describe, expect, it } from "vitest";
+import { describe, expect, it, vi } from "vitest";
 import * as env from "./env";
+import { afterEach, beforeEach } from "node:test";
 
 describe("env", () => {
   it("exports all of the correct constants", () => {
     expect(env).toBeDefined();
-    expect(Object.keys(env).length).toBe(7);
+    expect(Object.keys(env).length).toBe(4);
     expect(Object.keys(env)).toStrictEqual([
-      "isDevelopment",
-      "isProduction",
-      "isDebug",
       "USERFRONT_API_KEY",
       "USERFRONT_API_URL",
       "USERFRONT_API_VERSION",
       "USERFRONT_TENANT_ID",
     ]);
-    expect(env.isDevelopment).toBe(true);
-    expect(env.isProduction).toBe(false);
-    expect(env.isDebug).toBe(true);
   });
+
+  describe('build utils', () => {
```

Comment:
You can move these to `packages/utils.test.ts`.

The descriptions should read as sentences, for example, `it (should) ...`.

Thank you so much for your contributions!!! ❤️ Otherwise, LGTM!

Reply:
Moved tests to separate file.